jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/StepsRunner.java

```diff
@@ -358,22 +358,27 @@ private void buildAndCacheApplicationLayers() {
   private void buildImage() {
```

Comment:
Rename it to `buildImages()`? Also, rename other methods too (`pullBaseImage()` --> `pullBaseImages()`)?